app/models/course.rb

```diff
@@ -23,7 +23,7 @@ class Course < ActiveRecord::Base
   validates_presence_of :contact_first_name, :contact_last_name
   validates_presence_of :contact_email, :message => "contact email is required"
   validates_format_of :contact_email, :with => /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i, :message => "Invalid email"
-  validates_presence_of :contact_phone
+  #validates_presence_of :contact_phone
```

Comment:
I'm going to take this line out when I rebase my rails upgrade - we should prefer removing code rather than commenting it if it doesn't have instructive purpose, that's why we have things in version control.